src/_posts/platform/app/2000-01-01-scaling.md

```diff
@@ -48,7 +48,7 @@ data with our [CLI]({% post_url platform/cli/2000-01-01-start %}) with the
 following command:
 
 ```bash
-$ scalingo -a <appname> stats
+$ scalingo --app <appname> stats
```

Comment:
nitpick: we use `my-app` as the application name throughout the doc if you could also update that here :)

```suggestion
$ scalingo --app my-app stats
```